app/services/movie_service.rb

```diff
@@ -1,18 +1,46 @@
 class MovieService < ApiService
   def self.top_movies
-    endpoint = "https://api.themoviedb.org/3/discover/movie?api_key=#{ENV["movies_secret"]}&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false"
+    endpoint = "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV["movies_secret"]}&language=en-US&page=1"
+    #call an additional page if there are less than 40 movies
+    # each with object
+    # get_data inside enum
+    # endpoint = somestringpage=#{blockvariable with some integer}
+    # create MooVee PORO
+    # less than 40 movies?
+    # parse another endpoint page
+
     parsed_top_rated = get_data(endpoint)
+    #This can be 2 pages,
```

Comment:
This is making a lot more sense for me, now we just need to figure out how to access the top forty instead.